database/buildSchema/07B_template.sql

```diff
@@ -8,11 +8,12 @@ CREATE TYPE public.template_status AS ENUM (
 CREATE TABLE public.template (
     id serial PRIMARY KEY,
     name varchar,
-    code varchar NOT NULL,
+    code varchar NOT NULL UNIQUE,
     is_linear boolean DEFAULT TRUE,
     start_message jsonb,
     status public.template_status,
     submission_message jsonb DEFAULT '{"value": "Thank you! Your application has been submitted."}' ::jsonb,
```

Comment:
```suggestion
    submission_message jsonb DEFAULT '"Thank you! Your application has been submitted."' ::jsonb,
```
No need to write expressions in this old {value:...} style (and string literals are valid JSON).

Reply:
447877af67391a8a7fa0f6d883766a8b8e999bcb